.github/workflows/linux_job.yml

```diff
@@ -60,7 +60,7 @@ on:
         type: string
       docker-image:
         description: Identifies the Docker image by name.
-        default: "pytorch/conda-builder"
+        default: "pytorch/almalinux-builder"
```

Comment:
Let's make changes to `linux_job_v2.yml` instead, which already has these almalinux-based changes, so the only updates would be for the ROCm conditionalization.